src/ext/Firewall/ca/firewall.cpp

```diff
@@ -561,6 +549,16 @@ extern "C" UINT __stdcall ExecFirewallExceptions(
     pwz = pwzCustomActionData;
     while (pwz && *pwz)
     {
+        iTarget = fetUnknown;
+
+        ReleaseNullStr(attrs.pwzName);
```

Comment:
This isn't necessary -- the StrUtil functions reuse/reallocate memory.

Reply:
Updated the PR